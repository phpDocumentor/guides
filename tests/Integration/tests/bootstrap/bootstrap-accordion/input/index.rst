==============
Document Title
==============

..  accordion::
    :name: accordionExample

    ..  accordion-item:: Accordion Item #1
        :name: headingOne
        :header-level: 2
        :show:

        **This is the first item's accordion body.** It is shown by default, until the collapse plugin adds the
        appropriate classes that we use to style each element. These classes control the overall appearance,
        as well as the showing and hiding via CSS transitions.

        You can modify any of this with custom CSS
        or overriding our default variables. It's also worth noting that just about any HTML can go within
        the `.accordion-body`, though the transition does limit overflow.

    ..  accordion-item:: Accordion Item #2
        :name: headingTwo
        :parent: accordionExample
        :header-level: 2

        Placeholder content for this accordion, which is intended to demonstrate the .accordion-flush class.
        This is the second item's accordion body. Let's imagine this being filled with some actual content.

    ..  accordion-item:: Accordion Item #3
        :name: headingThree
        :parent: accordionExample
        :header-level: 2

        Placeholder content for this accordion, which is intended to demonstrate the .accordion-flush class.
        This is the third item's accordion body. Nothing more exciting happening here in terms of content, but
        just filling up the space to make it look,
        at least at first glance, a bit more representative of how this would look in a real-world application.
