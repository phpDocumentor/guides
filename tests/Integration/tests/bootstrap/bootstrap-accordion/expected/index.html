<!-- content start -->
                        <div class="section" id="document-title">
            <h1>Document Title</h1>
            <div class="accordion" id="accordion-parent-accordionExample">
            <div class="accordion-item">
            <h2 class="accordion-header" id="accordion-headingone-heading">
                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#accordion-headingone"
                        aria-expanded="true" aria-controls="accordion-headingone">
                    Accordion Item #1
                </button>
            </h2>
            <div id="accordion-headingone" class="accordion-collapse collapse show" aria-labelledby="accordion-headingone-heading"
                 data-bs-parent="#accordion-parent-accordionExample">
                <div class="accordion-body">

    <p><strong>This is the first item&#039;s accordion body.</strong> It is shown by default, until the collapse plugin adds the
appropriate classes that we use to style each element. These classes control the overall appearance,
as well as the showing and hiding via CSS transitions.</p>

    <p>You can modify any of this with custom CSS
or overriding our default variables. It&#039;s also worth noting that just about any HTML can go within
the <code>.accordion-body</code>, though the transition does limit overflow.</p>

                </div>
            </div>
        </div>        <div class="accordion-item">
            <h2 class="accordion-header" id="accordion-headingtwo-heading">
                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#accordion-headingtwo"
                        aria-expanded="false" aria-controls="accordion-headingtwo">
                    Accordion Item #2
                </button>
            </h2>
            <div id="accordion-headingtwo" class="accordion-collapse collapse" aria-labelledby="accordion-headingtwo-heading"
                 data-bs-parent="#accordion-parent-accordionExample">
                <div class="accordion-body">

    <p>Placeholder content for this accordion, which is intended to demonstrate the .accordion-flush class.
This is the second item&#039;s accordion body. Let&#039;s imagine this being filled with some actual content.</p>

                </div>
            </div>
        </div>        <div class="accordion-item">
            <h2 class="accordion-header" id="accordion-headingthree-heading">
                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#accordion-headingthree"
                        aria-expanded="false" aria-controls="accordion-headingthree">
                    Accordion Item #3
                </button>
            </h2>
            <div id="accordion-headingthree" class="accordion-collapse collapse" aria-labelledby="accordion-headingthree-heading"
                 data-bs-parent="#accordion-parent-accordionExample">
                <div class="accordion-body">

    <p>Placeholder content for this accordion, which is intended to demonstrate the .accordion-flush class.
This is the third item&#039;s accordion body. Nothing more exciting happening here in terms of content, but
just filling up the space to make it look,
at least at first glance, a bit more representative of how this would look in a real-world application.</p>

                </div>
            </div>
        </div></div>
    </div>
                    <!-- content end -->
